morph/react-native/get-block-name.js

```diff
@@ -57,6 +57,11 @@ const getGroupBlockName = (node, state) => {
     }
   }
 
+  if (hasProp(node, 'disabled')) {
+    const propNode = getProp(node, 'disabled')
+    node.disabled = propNode.value.value
```

Comment:
Ideally we wouldn't store stuff inside `node`. I know that below there's `backgroundImage` but that probably shouldn't be there either.

How about as a rule of thumb we only store things in `node` if they are going to be used across a few other places or if some children need it to perform more calculations?